WordPress/Classes/ViewRelated/Activity/JetpackActivityLogViewController.swift

```diff
@@ -32,4 +37,20 @@ class JetpackActivityLogViewController: BaseActivityListViewController {
 
         WPAnalytics.track(.activityLogViewed)
     }
+
+    private func configureBanner() {
+        containerStackView.addArrangedSubview(jetpackBannerView)
```

Comment:
Sorry I missed this earlier, it needs to be visible only on the WordPress app, and be under the `.jetpackPowered` feature flag.

Reply:
Oh yeah I'll fix it right away. I concluded that the subclass `JetpackActivityLogViewController` would only appear in WP but I was wrong and haven't tested it. Sorry for that!